### PR TITLE
Feat: Added home page

### PR DESCRIPTION
**Pull request by shuntzhere**
Added home page for my Quiz App.

Screenshot (28)

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  --bg-bright: #3cefff;
+  --bg-reef: #00f7ff;
+  --bg-dark: #4592af;
+
+  --color-white: #fdfdfd;
+  --color-black: #222222;
+  --color-shadow: #cfd9de;
+  /* font families */
+  /* font-family: "Lora", serif;
+  font-family: "Hind Siliguri", sans-serif;
```
Comment (SimranjitKaurMaan) [inline css/main.css:20]: If these styles are not required, don't forget to remove comments.

**Review by SimranjitKaurMaan — APPROVED** [review]
Looks good to me. One small suggestion.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  font-family: "Palanquin Dark", sans-serif;
+  */
```
Comment (krishanu2911) [inline css/main.css:22]: remove the comments if not needed

**Review by krishanu2911 — APPROVED** [review]
well written code just see that to remove unnecessary comment no changes requiered
nice design and well-written code!👍

File: css/main.css (hunk 1 above)
Comment (shuntzhere) [reply css/main.css:20]: Sure, so before the final merge I'll remove them. Thanks for the review.

File: css/main.css (hunk 2 above)
Comment (shuntzhere) [reply css/main.css:22]: Yes, I have put them there for references. They won't be there in the final dev branch. Thanks for reviewing.

**Pull request merged**